py/tests/algorithms/anomaly_likelihood_test.py

```diff
@@ -829,7 +837,24 @@ def testFilterLikelihoods(self):
     self.assertFalse(numpy.array_equal(l3a, l3b),
                      msg="Failure in case (iii), list 3")
 
+def testManySeeds():
 
+  AnomalyLikelihoodAlgorithmTest.GLOBAL_TEST_SEED=72
+  unittest.main()
+    
+#  for i in range(1,100):
+#    print("SEEEEEEEEEEEEEEEEEEEEEED:"+str(i))
+#    AnomalyLikelihoodAlgorithmTest.GLOBAL_TEST_SEED=i
+#    unittest.main()
+
+def testDeeply():
+  """
+  This function will execute tests many times and it will use really random seeds.
+  """
+  AnomalyLikelihoodAlgorithmTest.GLOBAL_TEST_SEED=0
+  for i in range(1,1000):
+    unittest.main()
```

Comment:
this will be slow. Probably enough to test once manually that this "many seeds" case works, and then remove it.

Reply:
yes of course, i wasn't planning to run that stuff on ordinary test.. this is just WIP, i will delete this when i finish it. But thanks for review anyway 👍